packages/ckeditor5-clipboard/tests/clipboard.js

```diff
@@ -833,3 +825,11 @@ describe( 'Clipboard feature', () => {
 		}
 	} );
 } );
+
+export function createDataTransfer( data ) {
+	return {
+		getData( type ) {
+			return data[ type ];
+		}
+	};
+}
```

Comment:
We don't export utils from tests. This is so small code that it could be copied to other tests (and AFAIR we have multiple copies of similar code). If we have some bigger utils that we use in many tests then we store them in the `tests/_utils` directory.

Reply:
Yup, I was wondering whether to copy it or export. Thanks!